### ISSUE TITLE
some questions about this framwork

### ISSUE THREAD

**Issue by tanjia123456** [Jeffyrao/neural-tweet-search#6]
Hello, can your algorithm be used for node classification? I think you quoted the article Convolutional neural networks for mesh-based parcellation of the cerebral cortex. Here I have a question. How should I visualize the results of node classification?

**Comment by jinfengr** [comment]
Hi @tanjia123456, actually this framework is a classification model. If you check out the loss function of the framework, it's a cross-entropy loss for classification. We simply adapt the framework to a ranking task (by ordering the prediction scores descendingly). But I have no idea about how it can be adapted to the cerebral cortex classification task as I am not familiar with the problem setting.

**Comment by tanjia123456** [comment]
Ok, any way, thank you very much! I think I have know how to solve my problem.
